controllers/openstackdataplanerole_controller.go

```diff
@@ -198,14 +200,15 @@ func (r *OpenStackDataPlaneRoleReconciler) Reconcile(ctx context.Context, req ct
 
 	if instance.Spec.DeployStrategy.Deploy {
 
-		r.Log.Info("Set DataPlaneRoleReadyCondition false")
-		instance.Status.Conditions.Set(condition.FalseCondition(dataplanev1beta1.DataPlaneRoleReadyCondition, condition.InitReason, condition.SeverityInfo, dataplanev1beta1.DataPlaneRoleReadyWaitingMessage))
+		r.Log.Info("Starting DataPlaneRole deploy")
+		r.Log.Info("Set ReadyCondition false")
+		instance.Status.Conditions.Set(condition.FalseCondition(condition.ReadyCondition, condition.InitReason, condition.SeverityInfo, dataplanev1beta1.DataPlaneRoleReadyWaitingMessage))
```

Comment:
```suggestion
		instance.Status.Conditions.Set(condition.FalseCondition(condition.ReadyCondition, condition.RequestedReason, condition.SeverityInfo, dataplanev1beta1.DataPlaneRoleReadyWaitingMessage))
```
Init is for Unknown: https://github.com/openstack-k8s-operators/lib-common/blob/master/modules/common/condition/conditions.go#L74